AFQ/api.py

```diff
@@ -139,18 +139,23 @@ def make_bundle_dict(bundle_names=BUNDLES, seg_algo="afq", resample_to=False):
 class AFQ(object):
     """
     """
+
     def __init__(self,
                  bids_path,
                  dmriprep='dmriprep',
                  segmentation='dmriprep',
                  seg_suffix='seg',
                  b0_threshold=0,
+                 target_b_val=None,
+                 target_b_sensitivity=50.0,
```

Comment:
I thought about this a bit more: instead of these, we could have `max_bval` and `min_bval` for this selection? Also, am I wrong to think that the selection here: https://github.com/yeatmanlab/pyAFQ/pull/283/files#diff-ae57f5dda2497f45a9b9955970ea8bddR466, throws out the b0? I don't think we want to do that. So we need to include all the volumes that have a b-value smaller than b0_threshold and all the volumes between max_bval and min_bval.